lib/Cargo.toml

```
@@ -55,6 +56,7 @@ nimiq-database = { workspace = true, optional = true }
 nimiq-genesis = { workspace = true, default-features = false }
 nimiq-jsonrpc-core = { workspace = true, optional = true }
 nimiq-jsonrpc-server = { workspace = true, optional = true }
+nimiq-hash = { workspace = true }
```

Let's keep the alphabetical order:
```suggestion
nimiq-genesis = { workspace = true, default-features = false }
nimiq-hash = { workspace = true }
nimiq-jsonrpc-core = { workspace = true, optional = true }
nimiq-jsonrpc-server = { workspace = true, optional = true }
```

---

I just can't do the alphabet in my head. Or at least I made an error at that time. ^^

---

Fixed.